modules/color.lua

```diff
@@ -305,14 +328,12 @@ function color.linear_to_gamma(r, g, b, a)
 
 	if type(r) == "table" then
 		local c = {}
-		for i = 1, 3 do
-			c[i] = convert(r[i] / 255) * 255
+		for i = 1, 4 do
+			c[i] = convert(r[i])
```

Comment:
this is suspect (was before, too!), alpha is intended to always be linear in all paths. probably my screwup, c[4] = r[4] was the intent.

Reply:
Ah, the original code makes more sense now. Fixed!